sdk/search/Azure.Search.Documents/src/AzureSearchDocumentsEventSource.cs

```diff
@@ -15,35 +18,94 @@ internal sealed class AzureSearchDocumentsEventSource : EventSource
         internal const int BatchSubmittedEvent = 2;
```

Comment:
We're not changing the EventSourceName?

Reply:
What do you suggest? This is the assembly leve EventSource for Azure.Search.Documents.
The name follows the guidance at https://azure.github.io/azure-sdk/dotnet_implementation.html#logging.

Reply:
I was hoping to make it batching specific, but skimming through the guidelines I see the one per library thing in which case what you have is correct.